backend/btrixcloud/crawlconfigs.py

```diff
@@ -238,17 +275,29 @@ async def add_crawl_config(
         config: CrawlConfigIn,
         org: Organization,
         user: User,
-        for_running_crawl=False,
     ):
         """Add new crawl config"""
+
+        # for now, to support frontend update logic
+        if config.oldId:
```

Comment:
I guess we can just remove this conditional now?

Reply:
removed